tests/everflow/test_everflow_testbed.py

```diff
@@ -15,7 +15,7 @@
 from tests.common.fixtures.ptfhost_utils import copy_arp_responder_py # noqa: F401, E501 lgtm[py/unused-import] pylint: disable=import-error
 
 pytestmark = [
-    pytest.mark.topology("t0", "t1", "m0")
+    pytest.mark.topology("t0", "t1", "m0", "t0-52")
```

Comment:
t0-52 is included in topo t0, no need to add it specifically.

Reply:
addressed comment